config/shapes/process_selection.py

```diff
@@ -20,6 +20,115 @@
+            "m_aiso_correction",
+        )
+    else:
+        weight = ("1.0", "m_aiso_correction")
+    return weight
```

Comment:
This is all super old legacy code ??

Reply:
Yes, we don't use them anymore, should I delete them?